blocks/curated-cards/curated-cards.css

```diff
@@ -18,15 +18,32 @@
 }
 
 .curated-cards .feature-card .feature-card-content {
+  display: flex;
   position: absolute;
   bottom: 20px;
   left: 20px;
   width: calc(100% - 40px);
   color: var(--color-white);
-  padding: 20px;
+  padding: 10px;
```

Comment:
can you please use spacing variable for this

Reply:
Hi @rrrragas We are holding off using some variables for now. As the client has sent some new vars which we will soon be incorporating. Also, these spacing values are not consistent for now in the designs.